packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Writers/CodeWriter.cs

```diff
@@ -269,11 +269,18 @@ public void WriteMethod(CSharpMethod method)
 
         public void WriteProperty(PropertyDeclaration property)
         {
+            if (!CurrentLine.IsEmpty)
+            {
+                WriteLine();
+            }
+
             if (property.Description is not null)
             {
-                WriteLine().WriteXmlDocumentationSummary(property.Description);
+                WriteXmlDocumentationSummary(property.Description);
             }
 
+            // TODO -- should write parameter xml doc if this is an IndexerDeclaration
+
```

Comment:
Can we create an issue for this and add the link here?

Reply:
Sure

Reply:
issue created: https://github.com/microsoft/typespec/issues/3276